web/src/search/queryTelemetry.tsx

```diff
@@ -8,15 +11,29 @@ export function queryTelemetryData(query: string, caseSensitive: boolean): { [ke
         empty: !query,
     }
 }
-
+function filterExistsInQuery(query: string, filterToMatch: string): boolean {
+    const parsedQuery = parseSearchQuery(query)
+    if (parsedQuery.type === 'success') {
+        const members = parsedQuery.token.members
+        for (const member of members) {
+            if (member.token.type === 'filter') {
+                const resolvedFilter = resolveFilter(member.token.filterType.token.value)
+                return resolvedFilter !== undefined && resolvedFilter.type === filterToMatch
```

Comment:
I just realised this is borked, sorry. Should be:

```suggestion
               if (resolvedFilter !== undefined && resolvedFilter.type === filterToMatch) {
                   return true
               }
```

(so that we keep iterating if the filter isn't matched)

Reply:
fixed